mlflow/paddle/__init__.py

```diff
@@ -0,0 +1,409 @@
+
+            y_train = y_train.reshape(-1, 1)
+            y_test = y_test.reshape(-1, 1)
+            return np.concatenate((X_train, y_train), axis=1), np.concatenate((X_test, y_test), axis=1)
+
+        class Regressor(paddle.nn.Layer):
+
+            def __init__(self):
+                super(Regressor, self).__init__()
+
+                self.fc = Linear(in_features=13, out_features=1)
+
+            @paddle.jit.to_static
+            def forward(self, inputs):
+                x = self.fc(inputs)
+                return x
+
+        model = Regressor()
+        model.train()
+        training_data, test_data = load_data()
+        opt = paddle.optimizer.SGD(learning_rate=0.01, parameters=model.parameters())
+
+        EPOCH_NUM = 10
+        BATCH_SIZE = 10
+
+        for epoch_id in range(EPOCH_NUM):
+            np.random.shuffle(training_data)
+            mini_batches = [training_data[k : k + BATCH_SIZE]
+                for k in range(0, len(training_data), BATCH_SIZE)]
+            for iter_id, mini_batch in enumerate(mini_batches):
+                x = np.array(mini_batch[:, :-1]).astype('float32')
+                y = np.array(mini_batch[:, -1:]).astype('float32')
+                house_features = paddle.to_tensor(x)
+                prices = paddle.to_tensor(y)
+
+                predicts = model(house_features)
+
+                loss = F.square_error_cost(predicts, label=prices)
+                avg_loss = paddle.mean(loss)
+                if iter_id%20==0:
+                    print("epoch: {}, iter: {}, loss is: {}".format(
+                        epoch_id, iter_id, avg_loss.numpy()))
+
+                avg_loss.backward()
+                opt.step()
+                opt.clear_grad()
+
+
+        mlflow.log_param('learning_rate', 0.01)
+        mlflow.paddle.log_model(model, "model")
+        sk_path_dir = './test-out'
+        mlflow.paddle.save_model(model, sk_path_dir)
+        print("Model saved in run %s" % mlflow.active_run().info.run_uuid)
+    """
+    import paddle
+
+    if os.path.exists(path):
+        raise MlflowException(
+            message="Path '{}' already exists".format(path), error_code=RESOURCE_ALREADY_EXISTS
+        )
+
+    os.makedirs(path)
+    if mlflow_model is None:
+        mlflow_model = Model()
+    if signature is not None:
+        mlflow_model.signature = signature
+    if input_example is not None:
+        _save_example(mlflow_model, input_example, path)
+
+    model_data_subpath = 'model'
+    output_path = os.path.join(path, model_data_subpath)
+
+    paddle.jit.save(pd_model, output_path)
+
+    conda_env_subpath = "conda.yaml"
+    if conda_env is None:
+        conda_env = get_default_conda_env()
+    elif not isinstance(conda_env, dict):
+        with open(conda_env, "r") as f:
+            conda_env = yaml.safe_load(f)
+    with open(os.path.join(path, conda_env_subpath), "w") as f:
+        yaml.safe_dump(conda_env, stream=f, default_flow_style=False)
+
```

Comment:
```suggestion
_log_pip_requirements(conda_env, path)
```

Could you add `_log_pip_requirements` here to log dependencies as `requirements.txt`?

Reply:
Sure! I'm adding it now.